### PR TITLE
mail carrier hood. AAAAAAAAAAAAH!!

### LINKED ISSUE #781: mail carrier winter coat has no hood
## Description
game breaking bug :sob:

### PR DESCRIPTION
**Pull request by Monotheonist**
## About the PR

fixes #781, gives the mail carrier a hood. Literally playable 😃 
This gives the mail carrier a hood to their coat.
## Why / Balance

It's crucial for the balance of the game, mail carriers everywhere have been deprived of their right for TOO LONG. I'm just kidding but I saw that I could fix this issue and so I did.
## Technical details

Currently YAML and sprites.
## Media

This is a hood, the sprite's in the diff. I don't got much else to say.
## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

No CL no fun.

### CODE REVIEW

**Comment by Monotheonist** [comment]
why the fuck did i name it that. oh my god. crying.

**Comment by deltanedas** [comment]
1. add a new hood entity, copy paste the logistics one and change the id/sprite/name
2. change the coats parent to `ClothingOuterWinterCoatToggleable`
3. add
```yml
- type: ToggleableClothing
  clothingPrototype: ClothingHeadHatHoodWinterMailCarrier
```
to the mail carrier winter coat

**Comment by Monotheonist** [comment]
I addressed it, my small yaml PR should be ready for your consideration @deltanedas

File: Resources/Prototypes/Nyanotrasen/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 1]
```diff
@@ -39,6 +39,8 @@
   name: mail carrier's winter coat
   description: It'll keep away the cold but not the dogs.
   components:
+    - type: ToggleableClothing
+     clothingPrototype: ClothingHeadHatHoodWinterMailCarrier #DeltaV - fixing the fact that it has no hood
```
Comment (deltanedas) [inline Resources/Prototypes/Nyanotrasen/Entities/Clothing/OuterClothing/wintercoats.yml]: add 1 space before clothingPrototype and move the comment to the line above

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Monotheonist** [comment]
There you go, are we good?

File: Resources/Prototypes/Nyanotrasen/Entities/Clothing/OuterClothing/wintercoats.yml (hunk 1 above)
Comment (Monotheonist) [reply Resources/Prototypes/Nyanotrasen/Entities/Clothing/OuterClothing/wintercoats.yml]: i did it 👍

**Review by deltanedas — DISMISSED** [review]

**Comment by Monotheonist** [comment]
The tests aren't testing 😿

**Comment by Monotheonist** [comment]
I'M FREE FROM THE CONSTANT TEST FAILS!!! @deltanedas @MilonPL

**Review by MilonPL — APPROVED** [review]

**Pull request merged**